### PR TITLE
[RFR] Index page rework 

### PR DESCRIPTION
**Pull request by enginehouse**
- Adding the journey as list items as currently mapped

### CODE REVIEW

File: app/server/api/static/apply-email-confirmation.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+contentType: page
+useComponent: true
+slug: apply-email-confirmation
+image: "/ui/img/content/registration-handshake-email.png"
```
Comment (Koded) [inline app/server/api/static/apply-email-confirmation.yml:5]: Do these need to be uploaded yet?

**Review by Koded — CHANGES_REQUESTED** [review]

File: app/server/api/static/apply-email-confirmation.yml (hunk 1 above)
Comment (enginehouse) [reply app/server/api/static/apply-email-confirmation.yml:5]: They are in the /ui/img/content/ folder so _should_ go up automatically. Are they missing from your build? @Koded

**Comment by enginehouse** [comment]
@Koded - could you take a look at this? It is getting a bit long and rambly due to its not being a particular user story.

File: app/server/api/static/apply-email-confirmation.yml (hunk 1 above)
Comment (Koded) [reply app/server/api/static/apply-email-confirmation.yml:5]: Ah yeah - can see they are in there. Ta.

**Review by Koded — APPROVED** [review]

**Pull request merged**